bucket/autohdr.json

```diff
@@ -0,0 +1,36 @@
+{
+    "homepage": "https://github.com/Codectory/AutoHDR",
+    "description": "Create profiles for your displays and apps to automatically switch the resolution, refresh rate and HDR state according to the app you're using",
+    "version": "1.7.19",
+    "license": "https://github.com/Codectory/AutoHDR/blob/main/LICENSE",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Codectory/AutoHDR/releases/download/1.7.19/Release_AutoHDR_1.7.19_x64.zip",
+            "hash": "13211a6cb0b6cc42754728cb991686582c5e6e5810e1af983bfd500c74681e96"
+        },
+        "32bit": {
+            "url": "https://github.com/Codectory/AutoHDR/releases/download/1.7.19/Release_AutoHDR_1.7.19_x86.zip",
+            "hash": "745ffd001d189b6d701647d76b2fef9318561eb83b4153035bf90cea2b36bb4a"
+        }
+    },
+    "pre_install": "if (!(Test-Path \"$persist_dir\\UserSettings.json\")) { New-Item -ItemType File \"$dir\\UserSettings.json\" | Out-Null }", 
+    "bin": "AutoHDR.exe",
```

Comment:
Does the binary take any commandline arguments?

Reply:
Not as far as I'm aware; nothing in the readme or any wiki available.

Think it's a simple app that reads config data .config and .json files in the same dir.

Reply:
In that case we can remove the bin entry. The shortcut is sufficient.

Reply:
Adjusted.